bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/api/BTypedesc.java

```diff
@@ -38,4 +39,10 @@
      * @return describing type
      */
     BType getDescribingType();
+
+    /**
+     * @param strand strand to be used to run the user-defined-type initialization code.
+     * @return instantiated object
+     */
+    Object instantiate(Strand strand);
```

Comment:
Do we have to pass in the strand? This is going to be generated method at runtime right?

Reply:
this method calls the `__init` of the newly created object, for that we need a strand, if not `start` keyword inside `__init` will not work.